Automattic/wp-calypso
# PostRevisions: Create explicit open/close methods for dialog

### Summary
Part of the follow up work of #19197, this PR aims to replace `togglePostRevisionsDialog` with the more explicit functions `closePostRevisionsDialog` & `openPostRevisionsDialog`.

### Testing
- Open a post in the editor that has history
- Click the 'history' button
- Note that the dialog opens
- Click close button in the dialog
- Note that the dialog closes
- Click the 'history' button to open again
- Use the <kbd>esc</kbd> key
- Note that the dialog closes again
- Click the 'history' button to open once more
- 'load' a revision
- Note that the dialog closes again
